content/php/concepts/math-functions/terms/is-infinite/is-infinite.md

```diff
@@ -0,0 +1,67 @@
+---
+Title: 'is_infinite()'
+Description: 'Evaluates whether a value is an infinite number or not.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Design'
+  - 'Web Development'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`is_infinite()`** function evaluates whether the value is infinite or not and returns a *boolean* value.
+
+
+## Syntax
+
+```pseudo
+$result = is_infinite(value);
+```
+
+Returns `1` if `value` is infinite, else returns `false/nothing`
+
+## Example
+
+The example below demonstrates the use of the `is_infinite()` function to check whether a number is infinite or not.
+
+```php
+<?php
+  $value = log(0);
+  $result = is_infinite($value);
+  echo $result; //this will output '1', indicating True
+?>
+```
+
+## Codebyte Example
+
+The codebyte below deploys the `is_infinite()` function on the values of `100` and `log(0)` to evaluate whether they are `infinite`:
+
+```codebyte/php
```

Comment:
code bytes are optional. Ideally all the examples included in an entry are serving a purpose and are not duplicative. This codebyte, is in part, restating the content from the previous example. I would suggest revising the example or deleting it.